### DISCUSSION TITLE
Add mass care of selected documents

### DISCUSSION THREAD

**Discussion by knas23** [LeoKlaus/Paperparrot#174] [Feature Requests]
Hi Leo, it would be very comfortable if there is a way to change the tag/document type of all select documents at once. 
Additional it would be perfect if I could select all documents of a specific filter at once. 

In generell thanks for the creat app!

**Comment by LeoKlaus** [comment]
Hey knas23, thanks for the suggestion!

It's actually already possible to set document type and correspondent or add tags when multiple documents are selected.
To do so, simply open the context menu (long press on iOS, right click on Mac) while multiple documents are selected and select the corresponding option.

A "select all" option is currently not implemented but I might add one at a later date.

**Comment by knas23** [comment]
Thx for the feedback. Sorry for the misunderstanding, my fault. I meant the function to remove the type/tag when multiple documents are selected. It would be great to remove the entrance tag after editing for multiple document at once. Or is it possible and I don't find the function...

**Comment by LeoKlaus** [comment]
Okay, I see.

No, that's not possible as of now.

I'll see how I can implement this.

IMO, the only intuitive way of handling this would be to show some sort of "x" next to the tag in the context menu when all selected documents have that tag.
